src/test/java/LoginTests.java

```diff
@@ -15,7 +15,7 @@ public static void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://apps.testpro.io/";
+        String url = "testpro.io/";
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221510530-6239fc43-cfec-41e7-a70b-79091ff976d6.png)

**Hint**   `String url = "https://testpro.io/";`